/**:
  ros__parameters:
    # If the number of consecutive invalid trajectory exceeds this threshold, the Diag will be set to ERROR.
    # (For example, threshold = 1 means, even if the trajectory is invalid, Diag will not be ERROR if
    #  the next trajectory is valid.)
    diag_error_count_threshold: 0

    display_on_terminal: false # show error msg on terminal

    thresholds:
      max_distance_deviation: 1.0
      max_reverse_velocity: 0.2
      max_over_velocity_ratio: 0.1
